README.md

```diff
@@ -42,6 +42,7 @@ Below are the rules supplied by this plugin and the information on passing optio
 | no-block-members               | Discourage use of members 'blockhash' & 'timestamp' (and alias 'now') of 'block' global variable | List of members to warn against   | ["blockhash", "timestamp"]           |       |
 | no-call-value                  | Discourage use of .call.value()()                                                                |                                   |                                      |       |
 | no-continue                    | Discourage use of 'continue' statement.                                                          |                                   |                                      |       |
+| no-assign-parameter            | Do not assign to parameters.                                                                     |                                   |                                      |       |
```

Comment:
`Disallow assigning to function parameters.`
(Please make this change everywhere in this PR where description is given)